.babelrc

```diff
@@ -1,7 +1,13 @@
 {
   // Javascript settings.
   "presets": [
-    "es2015",
+    ["env", {
+      "targets": {
+        "uglify": true,
```

Comment:
We have to set this because uglify doesn't support ES6. Though it doesn't really matter until we drop IE 11 support.